content/modules/redisgears/installing-redisgears.md

```diff
@@ -0,0 +1,144 @@
+
+## Minimum Requirements
+
+- Redis Enterprise 6.0.0 or later
+- The [cluster is setup]({{< relref "/rs/administering/cluster-operations/new-cluster-setup.md" >}}) and all of the nodes are joined to the cluster
+
+{{< note >}}
+For installing on Redis Enterprise Kubernetes deployments or Docker containers, you must use Ubuntu 18.04.4 LTS as the base OS because the RedisGears binaries are built using the binaries for that OS by [default](https://github.com/RedisLabs/redis-enterprise-k8s-docs#basic-installation), unless the RedisGears release specifies that there is also support for RHEL 7 docker images.
+{{< /note >}}
+
+## Installing RedisGears
+
+### Step 1: Install RedisGears dependencies
+
+On each node in the Redis Enterprise cluster:
+
+1. [Download](https://redislabs.com/download-center/modules/) the RedisGears Module - Dependencies Package from the Redis Enterprise Software section of the Downloads page.
+1. Copy the dependencies package to a node in your cluster.
+1. Run these commands as root:
+
+    ```src
+    # source /etc/opt/redislabs/redislabs_env_config.sh
+    # mkdir -p $modulesdatadir/rg/10000/deps/
+    # tar -xvf /home/ubuntu/redisgears-dependencies.linux-bionic-x64.1.0.0.tgz -C $modulesdatadir/rg/10000/deps
+    # chown -R $osuser $modulesdatadir/rg
+    ```
+
+{{< note >}}
+You must also run these commands on new nodes before you join the node to the cluster.
+{{< /note >}}
+
+### Step 2: Install the RedisGears module
+
+1. [Download](https://redislabs.com/download-center/modules/) the RedisGears Module.
+1. In the RS admin console, [add the RedisGears module]({{< relref "/modules/add-module-to-cluster.md" >}}) to the cluster.
+
+### Step 3: Create a Database and Verify the Installation
+
+1. Create a Redis Enterprise database [with the RedisGears module]({{< relref "/modules/add-module-to-database.md" >}}).
+1. To see that the database was created successfully and shards are running, run: `rladmin status`
+1. To see that RedisGears is running correctly, run: `redis-cli RG.PYEXECUTE "GearsBuilder().run()"`
+
+## Install the Write-Behind Recipe
+
+The Write-Behind recipe comes with two types of dependencies:
+
+- Drivers to connect to the backend database
+- Python libraries for the RedisGears Functions
+
+In most cases all of these can be provisioned to RedisGears before the Functions are uploaded.
+However, root access for the driver on the host is required in some cases, for example with Oracle drivers.
+
+### Step 1: Install Oracle driver (Optional)
+
+If you want to do write-behind with an Oracle database:
+
+1. [Download the Oracle driver](https://www.oracle.com/database/technologies/instant-client/linux-x86-64-downloads.html).
+1. On each RS node in the cluster, follow the installation instructions on the download page for the Oracle driver.
+
+### Step 2: Import requirements
+
+1. Install [Python 3](https://www.python.org/downloads/).
+1. To install the gears-cli, run: `pip install gears-cli`
```

Comment:
I suggest we also link to the pip repo https://pypi.org/project/gears-cli/ @bmansheim